example_simple_classification.py

```diff
@@ -0,0 +1,53 @@
+"""This file is used to create and compare two models on a particular dataset.
+It provides examples of reading from both csv and SQL Server. Note that this
+example can be run as-is after installing healthcare.ai. After you have
+found that one of the models works well on your data, move to Example2
+"""
+from healthcareai.simple_mode import SimpleDevelopSupervisedModel
+import pandas as pd
+import time
+
+
+def main():
+    t0 = time.time()
+
+    # CSV snippet for reading data into dataframe
+    dataframe = pd.read_csv('healthcareai/tests/fixtures/DiabetesClincialSampleData.csv', na_values=['None'])
+
+    # Drop columns that won't help machine learning
+    dataframe.drop(['PatientID', 'InTestWindowFLG'], axis=1, inplace=True)
```

Comment:
I'm confused about what `axis` is supposed to mean.  I thought `axis=0` means "columns" while `axis=1` means "rows".  This looks like you're deleting rows - is that not how this works?

Maybe it would be more clear to just do this (which I believe is equivalent):
```
del dataframe['PatientID']
del dataframe['InTestWindowFLG']
```